### PR TITLE
Add vLLM on Ray microservice

### PR DESCRIPTION
**Pull request by XinyaoWa**
## Description

Add vLLM on Ray microservice

## Issues

#194 

## Type of change
- [ ] New feature (non-breaking change which adds new functionality)

## Tests

tests/test_llms_text-generation_vllm-ray.sh

### CODE REVIEW

**Comment by XinyaoWa** [comment]
@chensuyue @lvliang-intel  This PR is ready to merge, please help to have a review~

File: tests/test_llms_text-generation_vllm-ray.sh [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    ## Build VLLM Ray docker
+    cd $WORKPATH/comps/llms/text-generation/vllm-ray/docker
+    docker build \
+        -f Dockerfile.vllmray  \
+        -t vllm_ray:habana --network=host ../../
+
+    ## Build OPEA microservice docker
+    cd $WORKPATH
+    docker build \
+        -t opea/llm-vllm-ray:latest \
```
Comment (chensuyue) [inline tests/test_llms_text-generation_vllm-ray.sh:20]: For test purpose, let name it as `opea/llm-vllm-ray:comps`

File: tests/test_llms_text-generation_vllm-ray.sh [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        -f comps/llms/text-generation/vllm-ray/docker/Dockerfile.microservice .
+}
+
+function start_service() {
+    export LLM_MODEL="facebook/opt-125m"
+    port_number=8006
+    docker run -d --rm \
+        --name="vllm-ray-service" \
```
Comment (chensuyue) [inline tests/test_llms_text-generation_vllm-ray.sh:28]: `--name=xxx` -> `--name=test-comps-xxx`  
We use this prefix for GenAIComps test containers, to make it separate with GenAIExamples CI test. We have limit HW resource, so we should make sure the name won't be conflict.

File: tests/test_llms_text-generation_vllm-ray.sh (hunk 1 above)
Comment (XinyaoWa) [reply tests/test_llms_text-generation_vllm-ray.sh]: updated.

File: tests/test_llms_text-generation_vllm-ray.sh (hunk 2 above)
Comment (XinyaoWa) [reply tests/test_llms_text-generation_vllm-ray.sh]: updated

**Review by lvliang-intel — APPROVED** [review]

**Review by chensuyue — APPROVED** [review]

**Pull request merged**